src/content/h1/tools/aether/readme.md

```diff
@@ -1,15 +1,219 @@
 
-# Tutorials
-* [YouTube - Basic Aether Tutorial - Halo Custom Edition](https://www.youtube.com/watch?v=x4cYQFW4Pxw)
-* [ModaWiki mirror - Aether](https://haloce3.com/tutorials/lighting/Aether.htm)
+# Aether usage tutorial
+**Aether - loading and exporting a BSP**
+
+Open up Aether and you'll see the main window. 
+
+<figure>
+  <a href="Aether_Main_Window.jpg">
+    <img src="Aether_Main_Window.jpg" alt="Aether's Main Window"/>
+  </a>
+  <figcaption>
+    <p>Aether's Main Window</p>
+  </figcaption>
+</figure>
+
+Go to File>New Project, or press the lightbulb button, and select the scenario file you are making lightmaps for. Then, save the Aether project file somewhere easy to remember. Aether will tick over for a bit depending on your computers speed and then load your project.
+
+Aether copies the information it needs into the project file so no alterations are made to your scenario or BSP files. This is the safest option, but does have the disadvantage that if you change something in your scenario (object placement for example) or recompile your BSP, your changes will *not* be reflected in your Aether project file. You will need to create a new Aether project to adjust for your changes.
+
+Depending on what you have in your map, the objects on the left will list all of your object types. Removing a type from the list will remove all instances of that type from the exported files. This can be handy if you have object types that do not cast a shadow or emit any light, and you don't want them in your exported files.
+
+<figure>
+  <a href="Aether_File_Loaded.jpg">
+    <img src="Aether_File_Loaded.jpg" alt="Aether loaded with a file"/>
+  </a>
+  <figcaption>
+    <p>Aether loaded with a file</p>
+  </figcaption>
+</figure>
+
+The BSP list selects which BSP you want to perfrom your actions on. With a BSP selected you have the option to Edit that BSPs' lightmap bitmap or export the data you need in 3ds MAX.
+
+If the Edit button is unavailable then your BSP has no lightmap bitmap associated with it, meaning you probably haven't debug lightmapped it yet.
+
+Hit the export button now.
+
+The options here should be pretty self explanatory, but you'll need at least the BSP and Lightmaps options selected for this tutorial. If you don't have all of your bitmaps exported already hit either All for all the bitmaps in all of the BSPs' shaders, or Diffuse only for just the diffuse textures. If you have objects in your map that you want to cast shadows or emit light then you need the Marker Export options checked too. Overwrite should be checked unless you've edited bitmaps for example, but you'll definately need it if you've recompiled your BSP as the lightmap UV's will be different. Hit export and Aether will output all of the info you need into your scenarios relative data directory.
+
+**3ds MAX - importing BSP**
```

Comment:
Headings should be done with `#` instead, e.g. `## Importing BSP`. This allows them to become part of the table of contents and receive ID attributes for anchor links.

Reply:
Should the individual steps of the tutorial be separate headings? I wasn't sure about that but I can do it that way.

Reply:
Yeah I think they should be headings, 1 level deeper than the tutorial's own heading level.